src/navbar/user-content.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { Grid } from '@material-ui/core'
+
+import { Gravatar } from 'navbar/gravatar'
+import { Email } from 'navbar/email'
+import { useStyles } from 'navbar/style'
+
+export const UserContent = () => <Grid item xs={4} lg={6}>
```

Comment:
Let's also handle the `sm` and `md` cases here.

Reply:
Done